src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckModule_srs.java

```diff
@@ -1,90 +0,0 @@
-package au.org.emii.gogoduck.worker;
-
-import java.io.File;
-import java.nio.file.Files;
-import java.util.ArrayList;
-import java.util.List;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import ucar.nc2.Attribute;
-import ucar.nc2.NetcdfFile;
-
-public class GoGoDuckModule_srs extends GoGoDuckModule {
-    private static final Logger logger = LoggerFactory.getLogger(GoGoDuckModule_srs.class);
-
-    private static final String srsVariables = "time,lat,lon,dt_analysis,l2p_flags,quality_level,satellite_zenith_angle,sea_surface_temperature,sses_bias,sses_count,sses_standard_deviation,sst_dtime";
```

Comment:
Have we catered for aggregating a specified list of variables - this should go to layer config